File: cpp/fibonacci.cpp
#include <iostream>

using namespace std;

long fib(long n);

long fib(long n) {
	// cout << "Fibonacci ke-" << n << endl;
	if ((n == 0) || (n == 1)) {
		return n;
	} else {
		return fib(n - 1) + fib(n - 2);
	}
}

int main() {
	long n, hasil;

	cout << "Nilai fibonacci ke :";
	cin >> n;

	hasil = fib(n);

	cout << "adalah " << hasil << endl;

	return 0;
}

File: Algo-2/latihan_rekursif/fibonacci.py
# Fungsi untuk mencari nilai fib ke N
def fibbonaci(n):
    if n <= 1:
        return n
    else:
        return fibbonaci(n - 1) + fibbonaci(n - 2)


n = int(input("Bilangan Real: "))
print(f"nilai fib ke {n} adalah {fibbonaci(n)}")
